core/event.js

```diff
@@ -96,6 +96,15 @@ if ( !CKEDITOR.event ) {
 			/**
 			 * Registers a listener to a specific event in the current object.
 			 *
+			 * **Note**: CKEditor's event system has a limitation that one function cannot be used as listener for the same event more than once.
```

Comment:
I would move it to the end of the description, because now it separates method description from main code sample. It would be more readable to add it at the end as this is a note which one should read after getting grasp of the main concept here (so how attaching listeners works).